harness/determined/pytorch/_pytorch_context.py

```diff
@@ -129,6 +129,12 @@ def get_lr_scheduler(self) -> Optional[pytorch.LRScheduler]:
             return self.lr_schedulers[0]
         return None
 
+    def get_use_amp(self) -> bool:
+        return self._use_amp
+
+    def get_amp_state(self):  # type: ignore
+        return apex.amp.state_dict()
```

Comment:
probably the caller of `get_amp_state()` should just call `apex.amp.state_dict()` anyway, since it's a global function in `amp` anyway, and has nothing to do with PyTorchTrialContext.